README.md

```diff
@@ -44,7 +44,19 @@ See a demo of how it works step by step [on my blog](https://cyber-engine.com/bl
 * Prepares, updates, reboots, and joins new nodes to an existing cluster with the `--add-nodes` flag, allowing you to grow your cluster as needed.
 
 # Usage
-Before using this project, see the [section on configuring your secrets files](#configuring-secrets-files).
+Before running any scripts, the secrets and environment variables files need to be updated with your configuration.
+
+### Add Proxmox Cluster User
+This project requires access to the proxmox cluster.
+On a proxmox server, run the commands in the shell to create a datacenter user.
+```shell
+pveum useradd terraform@pve -comment "Terraform User"
+pveum roleadd TerraformRole -privs "Pool.Audit, Pool.Allocate, VM.Allocate, VM.Config.Disk, Mapping.Use"
```

Comment:
The terraform provider suggests using more permissions
https://registry.terraform.io/providers/bpg/proxmox/latest/docs#api-token-authentication

`sudo pveum role add Terraform -privs "Datastore.Allocate Datastore.AllocateSpace Datastore.AllocateTemplate Datastore.Audit Pool.Allocate Sys.Audit Sys.Console Sys.Modify SDN.Use VM.Allocate VM.Audit VM.Clone VM.Config.CDROM VM.Config.Cloudinit VM.Config.CPU VM.Config.Disk VM.Config.HWType VM.Config.Memory VM.Config.Network VM.Config.Options VM.Migrate VM.Monitor VM.PowerMgmt User.Modify"`

Add `Pool.Audit` and `Mapping.Use` to those and it should be good 👍 
We'll also add